Spanner/src/Connection/Grpc.php

```diff
@@ -120,6 +122,16 @@ class Grpc implements ConnectionInterface
      */
     private $longRunningGrpcClients;
 
+    private function enableConnectionManagement($conf_path)
+    {
+        // TODO(ddyihai): move this function to GrpcTrait, if we are going
+        // to enable the grpc-gcp library for other apis.
+        $conf = new \Grpc\Gcp\ApiConfig();
+        $conf->mergeFromJsonString($string = file_get_contents($conf_path));
+        $config = new Config($conf);
```

Comment:
Browsing the [`Config.php`](https://github.com/GoogleCloudPlatform/grpc-gcp-php/blob/master/src/Config.php) source, I see that it uses semaphores. Many PHP users won't have that enabled by default, so we should add a check to `Config.php`, similar to what we do in the [auth client](https://github.com/google/google-auth-library-php/blob/master/src/Cache/SysVCacheItemPool.php#L122-L125).

Reply:
Thanks. I am copying the checks as in auth client: throw exception if it is not enabled.